%YAML:1.2
image_width: 1920
image_height: 1080
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  dt: d
  data: [505.812516, 0000.000000, 0512.664405,
         0000.000000, 506.481060, 0397.573724,
         0000.000000, 0000.000000, 0001.000000]
distortion_model: pinhole
distortion_coefficients:
  rows: 1
  cols: 5
  dt: d
  data: [   -0.216014866492617,     0.0891243427579082,    -0.000282632739383065,    -0.000420008453279489,    -0.0169601921763231]
rectification_matrix:
  rows: 3
  cols: 3
  dt: d
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 4
  cols: 3
  dt: d
  data: [-1720.654435, -0232.243636, 1203.142142, 30660.537932,
          0132.599515, -1921.888166, 0345.365066, 15515.012432,
          0000.115965, -0000.139809, 0000.983364, 00020.184879]
